models/topics.models.js

```diff
@@ -1,4 +1,12 @@
 const db = require('../db/connection');
+const fs = require('fs').promises;
+const test = require('../endpoints.json');
+
+exports.fetchEndpointsDescription = () => {
+  return fs
+    .readFile(`${__dirname}/../endpoints.json`, 'utf-8')
```

Comment:
same comments as above. This technically doesn't need to be a model and could just be required into your controller and sent from there, but this way is fine if you want to keep it like that